Documentation Edits

chriscavalluzzi opened the pull request:
Hello! I started going through the English documentation and making some tweaks. These include spelling/grammar fixes, making names/phrasing/capitalization consistent across different sections, formatting tweaks, adding links to other pages where appropriate, etc.

I've only done the intro page so far, but I wanted to pause and check if these edits are actually desired before I continued to other pages. I'm happy to continue or stop depending on if you think these changes are helpful.

(Quick note: There were some [hidden trailing backslashes](https://github.com/chriscavalluzzi/Butterfly/blob/186ba53a78b45087e12b5a7425514ecc946891c1/docs/src/content/docs/docs/v2/intro.mdx?plain=1#L76) that began displaying after I made edits, and I couldn't figure out what purpose they served, so I removed them in places where they were causing issues. But please let me know if those backslashes are important!)

CodeDoctorDE commented:
Hi feel free to make this pr and #824 ready. I can review it then and merge it on the release day

chriscavalluzzi commented:
Allllllrighty, this is probably a good stopping point. I changed a _LOT_, so feel free to change anything or let me know if you have any objections or concerns.

Some notes:
1. I moved the Migrating page to the bottom of the side bar, above the Tools section. Previously, it was positioned between some of the features, but I don't think that made sense. (Also, since the last version that was affected by migrations was released over 3 years ago, maybe it would be best to just remove the page instead?)
2. The Color Picker page probably needs a rework. I couldn't figure out if I wasn't looking in the right place, or if most of the buttons in the current screenshot have been removed.
3. I added a warning about packs being deprecated soon
4. I updated the notes about the Utilities features being moved to different locations. However, if you'd rather, those notes could just be removed instead.

CodeDoctorDE commented on docs/src/content/docs/docs/v2/areas.mdx:
```diff
@@ -1,33 +1,55 @@
+## Creating Areas
 
-While you can directly edit the content on the canvas, it is common that you would like to restrict the region in which you can modify the content, or that you would like ot be able to be able to quickly access a part of the document. For that, you have the <Monitor className="inline-icon"/> Area tool. As explained in the [introduction section](../), you might need to add the <Monitor className=" ...
+ With the Area tool selected, draw a rectangle anywhere on the canvas to define a new area. If you do not have the <Monitor className="inline-icon"/> Area tool on your toolbar, see [Customizing the Toolbar](../intro#customizing-the-toolbar).
```
Leading space, please remove it :)

chriscavalluzzi replied:
Thanks for catching these....... not sure how they got added, very weird!

CodeDoctorDE commented on docs/src/content/docs/docs/v2/areas.mdx:
```diff
@@ -1,33 +1,55 @@
+ With the Area tool selected, draw a rectangle anywhere on the canvas to define a new area. If you do not have the <Monitor className="inline-icon"/> Area tool on your toolbar, see [Customizing the Toolbar](../intro#customizing-the-toolbar).
 
-## More control
+ After creating an area, you can enter it using the <SignIn className="inline-icon"/> button in the [Context Menu](#the-context-menu).
```
Leading space here too

CodeDoctorDE commented on docs/src/content/docs/docs/v2/areas.mdx:
```diff
@@ -1,33 +1,55 @@
 
-Another option for you to use the area navigator is on the left menu, or on the butterfly menu if you are using a mobile platform.
+After entering an area, you can leave it by doing any of the following:
+- Tap the <Door className="inline-icon"/> button in the app bar
```
Please make a new line before an enumeration, looks better :)

CodeDoctorDE commented on docs/src/content/docs/docs/v2/intro.mdx:
```diff
@@ -1,96 +1,143 @@
-</Tabs>
+ </Tabs>
+
+  #### Customizing the Toolbar
```
Leading spaces

chriscavalluzzi replied:
Took this opportunity to fix all the tab-related formatting, should look much better now

CodeDoctorDE commented on docs/src/content/docs/docs/v2/pack.md:
```diff
@@ -2,6 +2,12 @@
 title: Pack
 ---
 
+:::caution[Planned deprecation]
```
I already wrote you on discord.
I think we could write it like: In-document pages will be deprecated soon. Make sure you save your packs globally.

chriscavalluzzi replied:
I like that phrasing! Updated.

CodeDoctorDE reviewed (changes requested):
Thank you very much for your contributions.
I have reviewed your changes and found some things I would change but other than that it looks good!
Having the migration page at the bottom of the page is a good idea.

With the packs system I think we need 1. a more detailed description and 2. we need to think about how we want to migrate to a "global pack system" without making a breaking change (I'm not really sure if I want to make a 3.0 now). We could add an automated migration and add an extra tab to allow people to export their deprecated document packs but then we would have that on all documents since it automatically adds the "Material" packs to all documents

chriscavalluzzi commented on docs/src/content/docs/docs/v2/areas.mdx:
```diff
@@ -1,33 +1,55 @@
 ![Area](area.png)
 
-Areas are a way to restrict the size of the canvas. You can create areas by using the [area tool](../tools/area). If you click on the background you can open the area list dialog using the context menu.
+Areas are a way to restrict the portion of the canvas that can be modified, allowing you to safely work on a specific region of your document without affecting other regions. You can create areas by using the [Area tool](../tools/area).
```
Updated! I couldn't figure out a smooth way to compare them to pages in other apps, but I think the new text is a good compromise

CodeDoctorDE reviewed (approved):
LGTM!

In the future we also need to change the intro page to the new 2.3 mobile ui
